libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/ThreadController.kt

```diff
@@ -103,6 +110,14 @@ object ThreadController {
         return JobControl(parentJob, CoroutineScope(parentJob + IODispatcher))
     }
 
+    /**
+     * Same as [getIOScopeAndRootJob], but using the Navigator dispatcher.
```

Comment:
Can we create documentation here independent of the IO scope

"Use this [JobControl] whenever interacting with the navigator.."